app/assets/stylesheets/base/fonts.scss

```diff
@@ -178,22 +178,89 @@ and (orientation : portrait) {
     h2 {
       font-size: 50px;
     }
-    p {
-        font-size: 35px;
-      }
+
     h3, h4, h5, h6 {
       font-size: 40px;
     }
     #signup {
```

Comment:
Why did you use an id here? If is just a normal style it's better to keep using a class instead of id.